dedibox-network/ip-failover/how-to/create-virtual-mac.mdx

```diff
@@ -35,42 +35,43 @@ If you start a virtual machine in bridge mode, protection will be activated on t
 You can add virtual MAC addresses known and allowed on the network to solve this problem.
 
 ### NATed network
-You share the same IP address for all your virtual machines with a NAT configuration. Your physical server and its virtual machines are seen as one single server. Only the MAC address of your physical network card is broadcasted on the network.
-With this configuration, no virtual MAC addresses are required. 
+
+In this network setup, all of your virtual machines are assigned the same public IP address through a NAT configuration. This arrangement makes your physical server and its virtual machines appear as a unified entity. The only MAC address that is broadcasted on the network is that of your physical network card.
+As a result, there is no need for individual virtual MAC addresses in this configuration.
 
 ## How to add a virtual MAC address
 
-1. Click **Server** > **Server list** to display a list of your physical machines. 
+1. Click **Server** > **Server list** to display a list of your physical machines.
 2. Select the server you want to configure and click **Manage**. The servers' status page displays.
 3. Scroll down to the network configuration section and click **+ Info** next to the failover IP you want to configure. The advanced configuration displays.
-4. Click **Add virtual MAC**. The virtual MAC configuration displays. 
-5. Click the cogwheel next to the IP address you want to configure. Then select the type of MAC address and click **Add ... virtual MAC address**.
+4. Click **Add virtual MAC**. The virtual MAC configuration displays.
+5. Click the cogwheel icon next to the IP address you want to configure. Then select the type of MAC address and click **Add ... virtual MAC address**.
     <Message type="note">
     Each MAC address is associated with manufacturer/organization. We provide virtual MAC addresses from the following Organizationally Unique Identifier (OUI):
     * OUI VMWare: `00:50:56`
     * OUI Xen: `00:16:3e`
     * OUI KVM: `52:54:00`
-    You can choose the OUI of your preference. They are technically interchangable. 
+    You can choose the OUI of your preference. They are technically interchangable.
 
     </Message>
 
 Once the virtual MAC is activated, use it to configure the network of your VM.
 
 ## How to delete a virtual MAC address
 
-You can remove the virtual MAC address associated to a failover IP if you do not require it any longer. 
+You can remove the virtual MAC address associated to a failover IP if you do not require it any longer.
 
 <Message type="tip">
   When you choose to delete a virtual MAC address on an IP address that shares its virtual MAC with other IPs, only the link between this specific IP address and the MAC is deleted, the other IP addresses are not updated.
```

Comment:
This sentence sounds a bit long and confusing, to me. 

Would something like this make sense? 

`If you delete a virtual MAC address that is shared by different IPs, this action will only affect the selected MAC address and IP. Any other IPs will remain as they are.`